### ISSUE TITLE
Please add Text weld feature 

### ISSUE THREAD

**Issue by AR4672** [svenhb/GRBL-Plotter#427]
Add text weld option in G Code creation (create text).

While creating gcode i found that alphabet gets detached in text mode, this creates a problem in vinyl cutting. 

![Capture](https://github.com/user-attachments/assets/52c489e1-6e93-40fe-8bdb-fc066da81aad)

**Comment by svenhb** [comment]
This not easy to implement.
I would use Inkscape to create the text,
- set "stroke" to black and "Fill" to none
- then "object to path"
- ungroup
- Edit node path to your whixhws
- copy whole drawing and paste to GRB-Plotter (with drag knife option enbaled)

**Comment by AR4672** [comment]
Thanks for the reply :)

Well, I have used this method and works nice, but I thought if create text would have the option to weld then it would have been an easy task to create precise gcode without any hassle.        
 
BTW that drag knife option is like a life saver for me.  

Actually, I have transformed a printer into cnc plotter but continuously facing precision issue, I thought it was a design flaw but later got to know about that offset setting, that's only present in your GRBL-Plotter.

All thanks to you ....
